### ISSUE TITLE
Lintian warning: hardening-no-bindnow usr/bin/garbd

### ISSUE THREAD

**Issue by ottok** [codership/galera#559]
While packaging galera-4 for Debian, I noticed Lintian complaning
> hardening-no-bindnow usr/bin/garbd

Full description at https://lintian.debian.org/tags/hardening-no-bindnow.html

Can this maybe be addressed directly upstream?

For more details, see https://wiki.debian.org/Hardening

Current hardening-check status:
```
# hardening-check -v /usr/bin/garbd
/usr/bin/garbd:
 Position Independent Executable: yes
 Stack protected: yes
 Fortify Source functions: yes (some protected functions found)
	unprotected: memmove
	unprotected: snprintf
	unprotected: memcpy
	unprotected: strncpy
	unprotected: strncat
	unprotected: poll
	unprotected: memset
	unprotected: strcpy
	protected: sprintf
	protected: snprintf
	protected: memcpy
	protected: fprintf
	protected: vsnprintf
	protected: syslog
	protected: strcat
 Read-only relocations: yes
 Immediate binding: no, not found!
 Stack clash protection: unknown, no -fstack-clash-protection instructions found
 Control flow integrity: unknown, no -fcf-protection instructions found!
```

**Comment by ottok** [comment]
Fixed in Debian via https://salsa.debian.org/mariadb-team/galera-4/-/commit/918044cf7b178a22bf3fc61a79317707494d6093. I will submit this upstream if somebody promises to merge it within a reasonable timeframe..

**Comment by ottok** [comment]
This was fixed in a85d90484614488944b0a4e6765fdf3c9e664b9b

```
# garbd --version
 INFO: 4.8.r

# hardening-check -v /usr/bin/garbd
/usr/bin/garbd:
 Position Independent Executable: yes
 Stack protected: yes
 Fortify Source functions: yes (some protected functions found)
	unprotected: memmove
	unprotected: memset
	unprotected: read
	unprotected: snprintf
	unprotected: strncat
	unprotected: poll
	unprotected: strncpy
	unprotected: memcpy
	unprotected: strcpy
	protected: vsnprintf
	protected: sprintf
	protected: fprintf
	protected: snprintf
	protected: syslog
	protected: memcpy
	protected: strcat
 Read-only relocations: yes
 Immediate binding: yes
```

Out of curiosity also tested [checksec.sh](https://github.com/slimm609/checksec.sh):

```
./checksec --file=/usr/bin/garbd --extended
RELRO           STACK CANARY      NX            PIE             SELFRANDO             Clang CFI            SafeStack            RPATH      RUNPATH	Symbols		FORTIFY	Fortified	Fortifiable	FILE
Full RELRO      Canary found      NX enabled    PIE enabled     No Selfrando          No Clang CFI found   No SafeStack found   No RPATH   No RUNPATH   4810) Symbols	  Yes	7		16		/usr/bin/garbd
```
